test/test_helper.rb

```diff
@@ -7,7 +7,7 @@
 require 'minitest/reporters'
 Minitest::Reporters.use! [Minitest::Reporters::SpecReporter.new]
 require 'database_cleaner'
-require "codeclimate-test-reporter"
+require 'codeclimate-test-reporter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.